app/controllers/withdrawals_controller.rb

```diff
@@ -0,0 +1,33 @@
+class WithdrawalsController < ApplicationController
+  before_action :set_shop, only: [:new, :create]
```

Comment:
Yeh I know it's just to be safe if new methods are added, but I'll change anyway